Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -35,6 +36,12 @@ BreakPoints breakpoints;
 MemChecks memchecks;
 PPCDebugInterface debug_interface;
 
+static int s_invalidate_cache_thread_safe;
+static void InvalidateCacheThreadSafe(u64 userdata, s64 cyclesLate)
+{
+  ppcState.iCache.Invalidate(userdata);
```

Comment:
To build on windows, you need to cast `userdata` to a u32.

    ppcState.iCache.Invalidate(static_cast<u32>(userdata)):